geometry_msgs/Point[] all_points
geometry_msgs/Point[] nearest_points
geometry_msgs/Point centroid
float32 max_dist
int32 slice_size
